mixins.scss

```diff
@@ -251,58 +270,61 @@
 			@content;
 		}
 	}
+	@else {
+		@warn "You are requesting an invalid breakpoint: `#{$point}`.";
```

Comment:
I would change that in another PR, because i will concentrate here only on the documentation